Plugins/CountWord/CountWord.cs

```diff
@@ -34,7 +35,8 @@ public PluginOutput Execute(PluginInput input)
                 while (index < str.Length && char.IsWhiteSpace(str[index]))
                     index++;
             }
-            return new PluginOutput($"The number of wrods in the text is: {wordCount} \r\nEnter new any text after the digit 6 to Count the number of words in the text.");
+            input.Callbacks.SendMessage($"The number of wrods in the text is: {wordCount} \r\nEnter new any text after the digit 6 to Count the number of words in the text.");
+            return;
```

Comment:
No need to return
You can remove this line completely
Also please check all the other places for this